torntools/scripts/content/items/ttItems.js

```diff
@@ -665,3 +669,97 @@ function addItemListener() {
 
 	doc.find("head").appendChild(script);
 }
+
+function showMissingPlushies() {
+	if (settings.pages.items.show_missing_plushies) {
+		let plushieSet = {
+			"Sheep Plushie": 186,
+			"Teddy Bear Plushie": 187,
+			"Kitten Plushie": 215,
+			"Jaguar Plushie": 258,
+			"Wolverine Plushie": 261,
+			"Nessie Plushie": 266,
+			"Red Fox Plushie": 268,
+			"Monkey Plushie": 269,
+			"Chamois Plushie": 273,
+			"Panda Plushie": 274,
+			"Lion Plushie": 281,
+			"Camel Plushie": 384,
+			"Stingray Plushie": 618,
+		};
+		let plushiesAvailable = userdata.inventory
+			.filter((x) => x.type === "Plushie")
```

Comment:
Parenthesis aren't really necessary here. Just saying :P

Reply:
Prettier adds them.

Reply:
Prettier adds them if you don't add them yourself.

Reply:
oh okay 👍